task2/Comment.php

```diff
@@ -0,0 +1,23 @@
+<?php
+include "User.php";
+
+class Comment
+{
+    private User $user;
+    private string $message;
+    public function __construct($user, $message)
+    {
+        $this->user = $user;
+        $this->message = $message;
+    }
+
+    public function getUser() : User
+    {
+        return $this->user;
+    }
+
+    public function getMessage()
```

Comment:
Не указан тип возвращаемого значения метода
```suggestion
    public function getMessage(): string
```